clients/imodels-client-authoring/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Change Log - @itwin/imodels-client-authoring 
+
+## 2.0.0
+
+Breaking changes:
+- Removed `FileHandler` interface and default `AzureFileHandler` implementation. Storage operations now use `@itwin/object-storage-core` and `@itwin/object-storage-azure` packages.
+- Removed internal code exports from the main `index.ts` file that should not be used directly by package users. The following components are no longer part of the public package API:
+  - Operation classes (`IModelOperations`, `ChangesetOperations`, etc.)
+  - Utility types (raw API response interfaces, etc.)
+  - Default IModelsClient option implementations (`AxiosRestClient`)
+  - Default iModels API parser class
+- Changed `_links` property type in entity interfaces from `Link` to `Link | null` as per iModels API definition.
+
+Please see the package documentation for an updated list of supported operations and entities: [link to docs](https://github.com/iTwin/imodels-clients/blob/main/docs/IModelsClientAuthoring.md).
```

Comment:
I don't know what is generally the preferable way to link other files, but wouldn't relative links work better? This link right now refers to a file in main branch, but you might not necessarily be viewing at it from main branch or on the main branch that file could end up being deleted in the main branch.

Reply:
We do not publish docs for now so relative links would not work. I am not sure I see that much benefit in publishing the docs so let's stick to this basic approach for now.